MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Indirect/Elwin.h

```diff
@@ -34,10 +34,12 @@ private slots:
   void unGroupInput(bool error);
   void saveClicked();
   void plotClicked();
+  void plotCurrentPreview();
 
 private:
   Ui::Elwin m_uiForm;
   QtTreePropertyBrowser *m_elwTree;
+  Mantid::API::MatrixWorkspace_sptr m_ElInputWS;
```

Comment:
We need to be careful when storing `shared_ptr` objects as it will increase the reference count on the workspace and keep it alive even if a user hits the delete button. Is it necessary to store the workspace?